interval.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type RefreshInterval time.Duration
+
+func (self RefreshInterval) String() string {
+	return time.Duration(self).String()
+}
+
+func (self *RefreshInterval) UnmarshalYAML(
+	unmarshal func(interface{}) error,
+) error {
+	i := 0
+	if err := unmarshal(&i); err == nil {
+		*self = RefreshInterval(time.Duration(i) * time.Second)
+		return nil
+	}
+	s := ""
+	if err := unmarshal(&s); err == nil {
+		i, err := strconv.ParseUint(s, 10, 64)
+		if err == nil {
+			*self = RefreshInterval(time.Duration(i) * time.Second)
+			return nil
```

Comment:
I am able to pass in `--new-file-check-interval=1` on the cmd line and r_s2 accepts it:

```
2015-07-30 00:41:49 DEBUG  remote_syslog.go:44 Evaluating globs every 1s
```

I think it's because we're returning here and not doing the check against `MIN_REFRESH_INTERVAL`.